serving/fn/model_process.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+import logging
+import joblib
+
+biomarkers = None
+pipeline = None
+model = None
+class ModelProcessor() :
+
+    def load(self, pipelinePath, modelPath) :
+        #load model from storage
+        global pipeline
+        global model
+
+        if pipeline == None:
+          logging.warning(f'**************  loading pipeline')
+          pipeline = joblib.load(pipelinePath)
+
+        if model == None:
+           logging.warning(f'**************  loading model')
+           model = joblib.load(modelPath)
+
+        return pipeline, model
+
+    def createDataFrame(self, jsondata) :
+        global biomarkers
+        if biomarkers == None:
+            biomarkers = ["HR","O2Sat","Temp","SBP","MAP","DBP","Resp","EtCO2","BaseExcess","HCO3","FiO2","pH","PaCO2","SaO2","AST","BUN","Alkalinephos","Calcium","Chloride","Creatinine","Bilirubin_direct","Glucose","Lactate","Magnesium","Phosphate","Potassium","Bilirubin_total","TroponinI","Hct","Hgb","PTT","WBC","Fibrinogen","Platelets","Age","Gender","Unit1","Unit2","HospAdmTime","ICULOS","isSepsis"]
```

Comment:
ideally we would want the data passed from FIHR to the model to exclude "Age", "Unit1", "Unit2", "HospAdmTime", "ICULOS", "Gender", "Bilirubin_direct", "TroponinI", "isSepsis" so we don't have to drop it here

Reply:
asked @dvanbalen to update the openapi spec to exclude these fields. once this is done and provided to the consumers, I will update the code that drops this fields in the api

Reply:
code changed

Reply:
pushed the new api spec and stubs